examples/update_flow_control/README.md

```diff
@@ -0,0 +1,108 @@
+# mender-client-python-example
+
+## Overview
+
+This example shows how to control the Update flow of the Mender client using the
+DBus API.
+
+At start time, it sets an Update Control Map that instructs the Mender client to
+"pause" on every possible state. These are `ArtifactInstall_Enter`,
+`ArtifactReboot_Enter`, and `ArtifactCommit_Enter`. Learn more about the Mender
+client states in the [States section from Mender Docs
+page](https://docs.mender.io/overview/state-script#states).
+
+Then the application waits for user input. The user can select a new Update
+Control Map to send to the Mender client. Each of the possible maps set a one of
+the previously mentioned states to "continue", which allows any ongoing update
+to progress.
+
+The code uses the Mender DBus API `io.mender.Update1` to set the Update Control
+Map in the Mender client. Refer to Mender Docs for more details on the actual
+[`io.mender.Update1.SetUpdateControlMap`
+method](https://docs.mender.io/development/device-side-api/io.mender.update1#setupdatecontrolmap)
+being used.
+
+## Usage
+
+See [main README](../../README.md) for instructions on how to install this
+package. Once installed, start this example with:
+
+```
+mender-update-flow-control
+```
+
+Which initializes the application, setting the "pause" all Update Control Map,
+and asks the user for further input with the following prompt:
+
+```
+Current map is:
+{
+    "priority": -1,
+    "states": {
+        "ArtifactInstall_Enter": {
+            "action": "pause"
+        },
+        "ArtifactReboot_Enter": {
+            "action": "pause"
+        },
+        "ArtifactCommit_Enter": {
+            "action": "pause"
+        }
+    },
+    "id": "01234567-89ab-cdef-0123-456789abcdef"
+}
+-----------------------
+What do you want to do?
+  0) Pause on all states
+  1) Continue with Installing new software
+  2) Continue with Rebooting
+  3) Continue with Committing new software
+  q) Quit
+
+```
+
+## Implementation
+
+The example application implements two threads that communicate with two queues;
+one thread responsible to manage the Mender DBus API communication, an another
```

Comment:
`one thread _is_ responsible`

Reply:
`for managing`

Reply:
`and another is responsible for dealing [...]`

Reply:
Sorry :weary: Thanks for your patience on these typos